iroh/src/getadd.rs

```diff
@@ -0,0 +1,132 @@
+use std::path::{Path, PathBuf};
+
+use anyhow::{Context, Result};
+use cid::Cid;
+use futures::Stream;
+use futures::StreamExt;
+use iroh_resolver::resolver::Path as IpfsPath;
+use iroh_resolver::{resolver, unixfs_builder};
+use iroh_rpc_client::Client;
+
+pub async fn get(client: &Client, ipfs_path: &IpfsPath, output: Option<&Path>) -> Result<()> {
```

Comment:
these should all be methods on `Client`

Reply:
Where you say make these methods of Client, do you mean we should move this code away from `iroh` into the rpc implementation, or do you want me to define a trait to that effect? I didn't write this code originally; I moved it and make it take client references instead of taking ownership, so now they look like methods.

Reply:
I could make them methods on `ClientApi` too.

Reply:
I've now reshuffled things, so when you're ready please take another look!